api-specs/test/types/links.raml

```diff
@@ -0,0 +1,39 @@
+#%RAML 1.0 Library
+usage: External link test type.
+
+uses:
+  scalars: scalars.raml
+  overrides: type-overrides.raml
+
+types:
+  LinkToTypeOnSameSite:
+    properties:
+      LinkToTypeOnSameSite:
+        description: A link pointing to a type the same microsite.
+        type: scalars.StringTestType
+  LinkToTypeOnAnotherSite:
+    properties:
+      LinkToTypeOnAnotherSite:
+        description: A link pointing to a type in another microsite.
+        type: overrides.TestTypeOnAnotherMicrosite
+  LinkToTypeOnExternalSite:
+    properties:
+      LinkToTypeOnExternalSite:
+        description: A link pointing to a type in an external website.
+        type: overrides.TestTypeOnAnExternalWebsite
+  LinkFromDescription:
+    description: |-
+      Link [ArrayTestType](/types/general#ctp:test:type:ArrayTestType) with `[ArrayTestType](/types/general#ctp:test:type:ArrayTestType)`
```

Comment:
This is not a link notation that we should use.   Use either URLs to headings or pure URNs.